app/build.gradle

```diff
@@ -68,8 +68,8 @@ dependencies {
     implementation 'com.mapbox.mapboxsdk:mapbox-android-plugin-annotation-v9:0.8.0'
 
     // Import mavsdk to control the drone
-    implementation 'io.mavsdk:mavsdk:0.3.1'
-    implementation 'io.mavsdk:mavsdk-server:0.3.1'
+    implementation 'io.mavsdk:mavsdk:0.8.0'
+    implementation 'io.mavsdk:mavsdk-server:0.8.0'
```

Comment:
Jonas Vauterin told me that they solved a bug in mavsdk-server:0.8.0 and now mavsdk-server:0.8.1 is the one most up to date.

Reply:
I bumped the version.
And for the camera parameters, I added a camera resolution field, but if you want something more, I can add it very easily